apps/challenges/views.py

```diff
@@ -132,3 +133,17 @@ def disable_challenge(request, pk):
         response_data = {
             'error': 'Sorry, you do not have permission to disable this challenge'}
         return Response(response_data, status=status.HTTP_401_UNAUTHORIZED)
+
+
+@api_view(['GET'])
+def get_past_challenges(request):
```

Comment:
`permission_classes` and `authentication_classes` are missing. Any specific reason?